### ISSUE TITLE
wpa suplicant is not working

### ISSUE THREAD

**Issue by danielo515** [kenfallon/fix-ssh-on-pi#2]
Hello,
thanks for this nice script and thanks for sharing it, it not only fulfils a purpose, but it is also full of interesting and nice tricks.

For some reason, the WPA supplicant is not working. I have it setup with my wifi settings, it gets copied but when I put the SD on the rpizero it does not joins the network.
I checked and the wpa supplicant seems to be copied to the root of the boot partition. Is that correct? isn't it supposed to be at `/etc/wpa_supplicant/wpa_supplicant.conf` ?

Thanks and regards

**Comment by danielo515** [comment]
Just checked that the correct thing to do is to put it on the boot partition and then it will be copied.
Then, what else can I check?

**Comment by kenfallon** [comment]
I checked the placement of the file, and it starts on the boot and then is automatically moved to /rootfs/etc/wpa_supplicant/wpa_supplicant.conf

I just tried this with a fresh install on 6 pi's and they all registered themselves on my wifi network. 

Can I suggest doing a manual install, then copying the wpa_supplicant.conf off the sdcard, then burn it again.

Check that the network router isn't blocking it as well.
